presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -88,6 +94,12 @@ public String getTableName()
         return tableName;
     }
 
+    @JsonProperty
+    public Optional<List<JdbcColumnHandle>> getColumns()
```

Comment:
add verification in `io.prestosql.plugin.jdbc.BaseJdbcClient#buildSql` that table handle's columns are same as the one passed explicitly

Reply:
I added it to JdbcRecordsetProvider